modules/ROOT/pages/design-common-problems-raml-10.adoc

```diff
@@ -1,4 +1,7 @@
 = Common Problems Found in RAML 1.0 API Specifications
```

Comment:
@kbostrom , the topic has been out since January and the content was reviewed by at least two PMs. They agreed to "problems".
Also, I've always felt that "issue" is an acceptable substitute (although too cringing for me) when someone is as sensitive as an overly ripe banana and their feeling "bruise" even with careful handling. "Problems" doesn't fling blame in anyone's face, but bluntly states the case: "You have to fix these things before moving on to grander vistas and skies of deeper blue. You're a rational creature, you're capable of this, and here is some help. Now, stand tall and have a go at it."